backend.py

```diff
@@ -41,7 +41,7 @@ def __repr__(self):
             self.direction, self.path, self.coordinates, self.lives, self.flags,
             self.damages, self.inactive)
 
-    def walk(self, distance, state, direction=None):
+    def walk(self, distance, state, direction=None, push_others=True):
```

Comment:
Udelas jeste tu schovanou metodu, jak jsme si ukazovali na srazu? `_move`, ktere se bude volat jak v `move`, tak v `walk`, nebo zatim je to konecna?

Reply:
jako jestli to chcete s tou schovanou metodou tak řekněte, mně se líbila víc tato verze, ale je mi to fuk.

Reply:
Mně se tahle verze taky líbila, ale ukazoval jsem toho pro úplnost víc :)